custom-metrics-autoscaling/direct-to-sd/sd_dummy_exporter.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"time"
+
+	"strings"
+
+	gce "cloud.google.com/go/compute/metadata"
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	monitoring "google.golang.org/api/monitoring/v3"
```

Comment:
We don't have complex examples in this example that require vendoring and you should be vendoring these stuff (e.g. golang.org/x/* pkgs). More of a sign of these stuff should not be in this repo.

Reply:
Added `go get` step to Dockerfile. I'm not sure if it resolves this. What else should be done here?